test/ibmq/websocket/test_websocket.py

```diff
@@ -29,13 +29,17 @@
     TOKEN_JOB_COMPLETED, TOKEN_JOB_TRANSITION, TOKEN_WRONG_FORMAT,
     TOKEN_TIMEOUT, websocket_handler)
 
+TEST_IP_ADDRESS = '127.0.0.1'
+TEST_WSS_PORT = '9876'
```

Comment:
Can you rename this variable? The difference is not that it is `wss` vs `ws`, it is that it signals an invalid port that we use for the `test_invalid_url` to fail.